src/fsharp/vs/IncrementalBuild.fs

```diff
@@ -1367,7 +1367,7 @@ type IncrementalBuilder(ctokCtor: CompilationThreadToken, frameworkTcImportsCach
     let assertNotDisposed() =
         if disposed then  
             System.Diagnostics.Debug.Assert(false, "IncrementalBuild object has already been disposed!")
-    let mutable referenceCount = 0
+    let referenceCount = ref 0
```

Comment:
Yes, but if you start at 1, then you can later make it illegal to increment from 0 to 1. (Because at 0, the object has already been disposed). This is just a preference.